app/decorators/controllers/hyrax/downloads_controller_decorator.rb

```diff
@@ -0,0 +1,30 @@
+module Hyrax
+  DownloadsController.class_eval do
```

Comment:
This is an unusual way to do this.  I like to do this same sort of thing by making a module and include it in the `DownloadsController`

in `config/initializers/hyku.rb` (or something) add:
`Hyrax::DownloadsController.include DownloadsControllerDecorator`

Reply:
Sounds good, I'll try this :+1: